File: src/main/java/com/jozufozu/exnihiloomnia/common/items/ExNihiloTabs.java
package com.jozufozu.exnihiloomnia.common.items;

import com.jozufozu.exnihiloomnia.common.blocks.ExNihiloBlocks;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class ExNihiloTabs
{
    public static final CreativeTabs BLOCKS = new CreativeTabs("exnihiloomnia.blocks")
    {
        @Override
        public ItemStack getTabIconItem()
        {
            return new ItemStack(ExNihiloBlocks.SIEVE);
        }
    };
    
    public static final CreativeTabs BARRELS = new CreativeTabs("exnihiloomnia.barrels")
    {
        @Override
        public ItemStack getTabIconItem()
        {
            return new ItemStack(ExNihiloBlocks.BARREL_WOOD);
        }
    };
    
    public static final CreativeTabs ITEMS = new CreativeTabs("exnihiloomnia.items")
    {
        @Override
        public ItemStack getTabIconItem()
        {
            return new ItemStack(ExNihiloBlocks.SIEVE);
        }
    };
}

File: src/main/java/com/jozufozu/exnihiloomnia/common/items/ExNihiloTabs.kt
package com.jozufozu.exnihiloomnia.common.items

import com.jozufozu.exnihiloomnia.common.blocks.ExNihiloBlocks
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

object ExNihiloTabs {
    val BLOCKS: CreativeTabs = object : CreativeTabs("exnihiloomnia.blocks") {
        override fun getTabIconItem(): ItemStack {
            return ItemStack(ExNihiloBlocks.SIEVE)
        }
    }

    val BARRELS: CreativeTabs = object : CreativeTabs("exnihiloomnia.barrels") {
        override fun getTabIconItem(): ItemStack {
            return ItemStack(ExNihiloBlocks.BARREL_WOOD)
        }
    }

    val ITEMS: CreativeTabs = object : CreativeTabs("exnihiloomnia.items") {
        override fun getTabIconItem(): ItemStack {
            return ItemStack(ExNihiloBlocks.SIEVE)
        }
    }
}
